screens/SettingsScreen/SettingsScreen.tsx

```diff
@@ -0,0 +1,653 @@
+import { Caption, Headline, Title } from "@components/Text"
+import { Ionicon } from "@components/common/Icons"
+import { AppStyles } from "@lib/AppColorStyle"
+import { useNavigation } from "@react-navigation/native"
+import { bool } from "aws-sdk/clients/signer"
+import React, { useState } from "react"
+import { ScrollView, StyleSheet, Switch, View } from "react-native"
+import { Divider } from "react-native-elements"
+
+export const SettingsScreen = () => {
+  // List of attendees
+
+  const navigation = useNavigation()
+  /**
+   * Creates a switch for a given setting, and uses states to keep track of its current on-off status.
+   *
+   * @param setting: A given setting you want to keep track of.
+   * @returns A pre-built Switch component for this specific setting.
+   */
+  const SwitchComponent = (setting: any) => {
+    const [switchState, setSwitchState] = useState(setting.state)
```

Comment:
Should be a setValue prop instead of a direct useState, since state can be volatile